packages/dapper/src/errors.ts

```diff
@@ -0,0 +1,15 @@
+export class DapperError extends Error {
```

Comment:
I remember there being some wonkiness when it comes to extending (and appropriately catching) custom errors in JS/TS. As error handling isn't implemented yet, I don't think it's a major issue, but just noting down for when we're adding error handling as we'll probably want to write tests for that.

Reply:
https://app.clickup.com/t/20413243/TS-307